src/twisted/internet/test/test_inlinecb.py

```diff
@@ -145,3 +147,181 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         self.assertIn("in erroring", f.getTraceback())
+
+
+
+class TranslateMe(Exception):
+    """
+    Sample exception type.
+    """
+
+
+
+class TranslateResult(Exception):
+    """
+    Sample exception type.
+    """
+
+
+
+class DontFail(Exception):
+    """
+    Sample exception type.
+    """
+
+    def __init__(self, actual):
+        Exception.__init__(self)
+        self.actualValue = actual
+
+
+
+class CancellationTests(TestCase):
+    """
+    Tests for cancellation of L{Deferred}s returned by L{inlineCallbacks}.
+    For each of these tests, let:
+        - G be a generator decorated with C{inlineCallbacks}
```

Comment:
This is informative notation! :100:

It might be nice to wrap the capital letters in `C{}`.